client/main/main.js

```diff
@@ -164,7 +164,7 @@ $(document).ready(function() {
     var ribbon = new Ribbon({ el : '.silky-ribbon', model : ribbonModel });
     var backstage = new Backstage({ el : "#backstage", model : backstageModel });
 
-    notifications = new Notifications({ el : '#selector-here'});
+    notifications = new Notifications($('#notifications'));
 
```

Comment:
I tried doing that but it didn't work because notifications isn't a view like the others.